### PR TITLE
Add new confirmation screen for pending WorldPay payment

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-1013

This adds a new completion screen for when a WorldPay payment is pending

<details> 

<summary> Screen </summary>

Screenshot 2020-05-05 at 12 48 45

</details>

<details> 

<summary> Email </summary>

Screenshot 2020-05-05 at 12 37 32

</details>

### CODE REVIEW

**Comment by cintamani** [comment]
SonarCloud doesn't like the email template, as usual, so ignoring it

File: config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  waste_carriers_engine:
+    registration_received_pending_worldpay_payment_forms:
+      new:
+        title: "We’re processing your payment"
+        heading: "We’re processing your payment"
+        highlight_text: "Please use this registration number if you contact us:"
+        paragraph_1: "We've sent a confirmation email to %{email}."
+        subheading_1: "We have received your application"
+        paragraph_2: "We’re currently processing your payment and will let you know when it has been received."
+        paragraph_3: "Your registration will not be active until we have confirmed your registration"
```
Comment (irisfaraway) [inline config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml:11]:
```suggestion
        paragraph_3: "Your registration will not be active until we have confirmed your registration."
```

File: spec/factories/order.rb [hunk 2]
```diff
@@ -10,6 +10,12 @@
       total_amount { order_items.sum { |item| item[:amount] } }
     end
 
+    trait :has_pending_worldpay_status do
+      has_required_data
+
+      worldPayStatus { "SENT_FOR_AUTHORISATION" }
```
Comment (irisfaraway) [inline spec/factories/order.rb:16]:
```suggestion
      world_pay_status { "SENT_FOR_AUTHORISATION" }
```

File: config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml (hunk 1 above)
Comment (cintamani) [reply config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml:11]: there is no dot in the wireframe for this

**Comment by cintamani** [comment]
Yes, I am all in favour of splitted controllers. I hate to deal with too many ifs and maintenance of the code will improve.

File: config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml (hunk 1 above)
Comment (irisfaraway) [reply config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml:11]: Sorry, didn't realise this was the bit of text in the panel! I thought it was just a regular paragraph. Ignore me!

**Pull request merged**